backend/src/main/resources/application-local.yaml

```diff
@@ -4,7 +4,7 @@ spring:
     username: test
     password: test
   flyway:
-    locations: classpath:db/migration,classpath:db/data/real,classpath:db/data/synthetic
+    locations: classpath:db/migration,classpath:db/data/real,classpath:db/data/synthetic,classpath:db/data/e2e
```

Comment:
Why adding this? We said that we want to decide locally with which profile to run the backend and with this change and only the local profile, we will always be importing also the e2e test data. I guess you have it set locally this way and commit the change without realizing?

Reply:
Oh yes i committed that accidentally. Needed to add that part locally so I could run the e2e tests. 

Also I'm not sure what the advantage of not having the e2e data in the local database is. We do not import any additional data there and we need it to run e2e tests.